radix-engine/src/kernel/heap.rs

```diff
@@ -100,6 +104,7 @@ impl Heap {
         Ok(())
     }
 
+    #[trace_resources]
```

Comment:
Also, every pub method.

Reply:
Same as for call_frame.rs, I removed tracing from heap.rs.